Mathlib/FieldTheory/RatFunc/AsPolynomial.lean

```diff
@@ -223,6 +223,49 @@ instance : Valued (RatFunc K) ℤₘ₀ := Valued.mk' (idealX K).valuation
 theorem WithZero.valued_def {x : RatFunc K} :
     @Valued.v (RatFunc K) _ _ _ _ x = (idealX K).valuation x := rfl
 
+/- (Degree of a rational function field).
```

Comment:
Write /-- instead of /-, as this will generate comments on the Mathlib documentation website.